src/connector/src/source/mqtt/enumerator/mod.rs

```diff
@@ -48,44 +43,20 @@ impl SplitEnumerator for MqttSplitEnumerator {
         context: SourceEnumeratorContextRef,
     ) -> ConnectorResult<MqttSplitEnumerator> {
         let (client, mut eventloop) = properties.common.build_client(context.info.source_id, 0)?;
-
         let topic = properties.topic.clone();
```

Comment:
If wildcards are not supported, whether to add a check to `MqttSplitEnumerator` to display the user should not use a topic with wildcards

Reply:
Synchronously modify comments on `MqttProperties#topic`